backend.py

```diff
@@ -242,6 +232,12 @@ def name(self):
     def __repr__(self):
         return "<{} {} {}>".format(self.name, self.priority, self.rotation)
 
+    def apply_effect(self, robot):
```

Comment:
Ješte jsem odhalila chybičku. Tady musí být stejné volání jako u `MovementCard`, aby se dalo kdykoliv zavolat odehrání karty bez nutnosti řešit typ karty.
```suggestion
    def apply_effect(self, robot, state):
```